Parser/tokenizer/helpers.c

```diff
@@ -129,7 +132,13 @@ _PyTokenizer_warn_invalid_escape_sequence(struct tok_state *tok, int first_inval
             /* Replace the SyntaxWarning exception with a SyntaxError
                to get a more accurate error report */
             PyErr_Clear();
-            return _PyTokenizer_syntaxerror(tok, "invalid escape sequence '\\%c'", (char) first_invalid_escape_char);
+
+            return _PyTokenizer_syntaxerror(tok,
+                "\"\\%c\" is an invalid escape sequence. "
+                "Such sequences will not work in the future. "
```

Comment:
Currently, it already raises a SyntaxError. I don't think that "in the future" is correct here.

Reply:
Fair point. I've removed 'Such sequences will not work in the future.' from both the `SyntaxError`s and tests for the `SyntaxError`s.

I'm curious how a `SyntaxError` can be raised when `SyntaxWarning`s are all I get? Not sure what the trigger is.